### ISSUE TITLE
A way to invoke functions, wrapped in `PythonFunctionContainer`

### ISSUE THREAD

**Issue by lostmsu** [losttech/Gradient#19]
In Keras you can access activation functions like this: 
`var activation = tf.keras.activations.some_activation_fn;`

In Preview 6 you need to cast it to `dynamic`, to apply it to a tensor, which is quite ugly:
`((dynamic)activation)(tensor1, tensor2)`.

Ideally, I'd like to be able to call `activation.Invoke(tensor1, tensor2)`
